### ISSUE TITLE
Render using explicit each serializer test failing sometimes

### ISSUE THREAD

**Issue by edwardloveall** [rails-api/active_model_serializers#951]
The test added in #949 [here](https://github.com/rails-api/active_model_serializers/pull/949/files?diff=unified#diff-ef7576d68f65ad1dc77b2535b639f048R117) is often failing (about 50% of the time). My thought is that it has something to do with all the instance variables in many other tests. This is a common error to receive:

```diff
--- expected
+++ actual
@@ -1 +1 @@
-"{\"id\":1,\"title\":\"New Post\",\"body\":\"Body\",\"comments\":[{\"id\":1,\"body\":\"ZOMG A COMMENT\"}],\"blog\":{\"id\":999,\"name\":\"Custom blog\"},\"author\":{\"id\":1,\"name\":\"Joao Moura.\"}}"
+"{\"id\":\"1\",\"type\":\"posts\",\"attributes\":{\"title\":\"New Post\",\"body\":\"Body\"},\"comments\":[{\"id\":1,\"body\":\"ZOMG A COMMENT\"}],\"blog\":{\"id\":999,\"name\":\"Custom blog\"},\"author\":{\"id\":1,\"name\":\"Joao Moura.\"}}"
```

And this one:

```diff
--- expected
+++ actual
@@ -1 +1 @@
-"{\"id\":1,\"title\":\"New Post\",\"body\":\"Body\",\"comments\":[{\"id\":1,\"body\":\"ZOMG A COMMENT\"}],\"blog\":{\"id\":999,\"name\":\"Custom blog\"},\"author\":{\"id\":1,\"name\":\"Joao Moura.\"}}"
+"{\"id\":1,\"title\":\"New Post\",\"body\":\"Body\",\"comments\":[{\"id\":1,\"body\":\"ZOMG A COMMENT\"}],\"blog\":{\"id\":999,\"name\":\"Custom blog\"},\"author\":{\"id\":1,\"name\":\"Steve K.\"}}"
```

It's been very difficult to track down this error so far. It's repeatable if I use the seed `5975`:

```
rake TESTOPTS='--seed=5975'
```

**Comment by bf4** [comment]
@edwardloveall See my PR into your PR https://github.com/edwardloveall/active_model_serializers/pull/1

**Comment by joaomdmoura** [comment]
Glad that you guys made it :smile: 
I saw it was already merged and the #949 was updated so I'm closing this one
